setup.py

```diff
@@ -44,6 +45,7 @@
     author="Elyra Maintainers",
     license="Apache License Version 2.0",
     data_files=[('etc/jupyter/jupyter_notebook_config.d', glob(auto_extension_path)),
+                ('etc/jupyter', glob(notebook_config_path)),
```

Comment:
this will replace any configuration/customization that the user might have done on their notebook as the `jupyter_notebook_config.py` is widely used

Reply:
actually I tested that, it will not override or replace any settings in ~/.jupyter/jupyter_notebook_config.py`